apps/dashboard/src/routes/(search-pages)/resource/[resource]/+layout.ts

```diff
@@ -55,7 +57,16 @@ export const load: LayoutLoad = async ({ params }) => {
       ? getRedeemableTokens(transformedResource)
       : Promise.resolve(undefined)
 
+  const gateway = GatewayApiClient.initialize({
```

Comment:
I'd move this to `$lib/gateway.api` and change `applicationDappDefAddress`. 

```
// src/lib/gateway.ts
export const gateway = GatewayApiClient.initialize({
    networkId: CURRENT_NETWORK.id,
    applicationName: 'Radix Dashboard',
    applicationDappDefinitionAddress: CURRENT_NETWORK.dashboardDappAddress
  })
 ```

Reply:
changed to `callApi`

Reply:
😢